pkg/hs/urbit-king/lib/Urbit/Vere/Term/Logic.hs

```diff
@@ -111,8 +113,8 @@ drawState :: St -> [Ev]
 drawState St{..} = hist <> out <> cur <> spin
   where
     hist = drawHistory <$> toList sHistory
-    out  = if null sLine   then [] else [EvEdit sLine]
-    cur  = if 0 == sCurPos then [] else [EvMove $ fromIntegral $ sCurPos]
+    out  = if null sLine       then [] else [EvEdit sLine]
```

Comment:
isn't this just `map EvEdit sLine`?

Reply:
No? `sLine` isn't a list, so it doesn't let me `map` over it. Also, we want to explicitly produce `[]` for the null/empty string case, not `[EvEdit null]` or w/e.